src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -17,6 +17,8 @@ $reply-icon-radius: 20px;
   flex-direction: column;
   margin-bottom: $spacing--lg;
 
+  content-visibility: auto;
```

Comment:
@mike-lvov FYI: This seems to cause performance issues and pins my GPU at 50%+ processing even when just idling. Commenting out this line removes the issues. See further details in this thread:

- https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1626669763475800

Before commenting out the line:
![image](https://user-images.githubusercontent.com/753891/126104439-0a712b34-8b9a-41df-870c-98dcc2f0a64d.png)

Each of those little marks zoomed way in:
![image](https://user-images.githubusercontent.com/753891/126104451-3eafed4f-784f-4178-bd86-06952465ae40.png)

Zooming out a little we can see it runs every 15ms or so:
![image](https://user-images.githubusercontent.com/753891/126104459-51e3d4ea-fd2a-48fa-a584-bd40c482d09a.png)

After commenting out the line, when GPU usage returns to normal/barely registering:
![image](https://user-images.githubusercontent.com/753891/126104523-9a307e62-dd9e-48c0-9b3c-cccfa390e74f.png)